vtex/andromedocs
# Use andromedocs-theme

#### What is the purpose of this pull request?
Configures the /docs folder to use andromedocs-theme

#### What problem is this solving?
Resolve #24 

#### How should this be manually tested?
run (in the /docs folder)
> gatsby develop 
    
or
> yarn workspace docs develop 

You can also create an mdx file or a folder with this mdx file, to see it rendered with the theme. In addition you can apply the "shadowing" of either the theme-ui or another component

#### Types of changes

#### How does this PR make you feel? [:link:](http://giphy.com/)
<img src="https://media1.tenor.com/images/f94949eab0905710b4f2ea4666631c2c/tenor.gif?itemid=4062409" alt="happy" width="400"/>
